results.html

```diff
@@ -17,10 +17,6 @@
 <link href="https://fonts.googleapis.com/css?family=Alegreya|Alegreya+Sans|Cabin|Inconsolata|Merriweather|Merriweather+Sans|Nunito|Nunito+Sans|Pacifico|Quattrocento|Quattrocento+Sans|Quicksand|Roboto|Roboto+Mono|Roboto+Slab|Rubik|VT323&display=swap" rel="stylesheet">
 <link href="https://fonts.googleapis.com/css?family=Iceland&display=swap" rel="stylesheet">
 <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"> -->
-
-<!-- Momment.js -->
-<script src="https://cdn.jsdelivr.net/momentjs/2.12.0/moment.min.js"></script>
```

Comment:
We need this link  to the CDN for the time function I added to app.js to work. Did you move it to a new location I'm not seeing?